BatchManagement.json

```diff
@@ -0,0 +1,1618 @@
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/BatchAccountRegenerateKeyParameters"
+            },
+            "description": "The type of key to regenerate."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation was successful. The response contains the Batch account keys.",
+            "schema": {
+              "$ref": "#/definitions/BatchAccountKeys"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Batch/batchAccounts/{accountName}/listKeys": {
+      "post": {
+        "tags": [
+          "BatchAccount"
+        ],
+        "operationId": "BatchAccount_GetKeys",
+        "x-ms-examples": {
+          "BatchAccountGetKeys": { "$ref": "./examples/BatchAccountGetKeys.json" }
+        },
+        "summary": "Gets the account keys for the specified Batch account.",
+        "description": "This operation applies only to Batch accounts created with a poolAllocationMode of 'BatchService'. If the Batch account was created with a poolAllocationMode of 'UserSubscription', clients cannot use access to keys to authenticate, and must use Azure Active Directory instead. In this case, getting the keys will fail.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation was successful. The response contains the keys of the Batch account.",
+            "schema": {
+              "$ref": "#/definitions/BatchAccountKeys"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Batch/batchAccounts/{accountName}/applications/{applicationId}/versions/{version}/activate": {
+      "post": {
+        "tags": [
+          "ApplicationPackage"
+        ],
+        "operationId": "ApplicationPackage_Activate",
```

Comment:
As we use the `NOUN` part of `NOUN_VERB` pattern in `operationId` in the grouping of operation. Currently, we have a model named as `NOUN` that'll be auto resolved by autorest to resolve naming conflicts. 

Reference: https://travis-ci.org/Azure/azure-rest-api-specs/jobs/254658613#L916

Please review 18 of these [`OperationIdNounConflictingModelNames`] warning in the build [here](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/254658613#L646)

Please make sure that is what you expect in SDK or we need to make them differ here.